tests/layout_visualization_test.py

```diff
@@ -0,0 +1,33 @@
+
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+import floris.tools.layout_visualization as layoutviz
+from floris.tools.floris_interface import FlorisInterface
+
+
+TEST_DATA = Path(__file__).resolve().parent / "data"
+YAML_INPUT = TEST_DATA / "input_full.yaml"
+
+
+def test_get_wake_direction():
+    # Turbine 0 wakes Turbine 1 at 270 degrees
+    assert np.isclose(layoutviz.get_wake_direction(0, 0, 1, 0), 270.0)
+
+    # Turbine 0 wakes Turbine 1 at 0 degrees
+    assert np.isclose(layoutviz.get_wake_direction(0, 1, 0, 0), 0.0)
+
+    # Winds from the south
+    assert np.isclose(layoutviz.get_wake_direction(0, -1, 0, 0), 180.0)
+
+def test_plotting_functions():
+
+    fi = FlorisInterface(configuration=YAML_INPUT)
```

Comment:
Include test for `plot_turbines_rotors`

Reply:
I've added this test, but it would also be good to test `plot_farm_terrain`.
Also, I think I know the answer to this, but is there a better test available? It's good to run these functions, but as is this is only testing that the inputs are of the correct general data type.

Reply:
Yeah, this was a puzzle for me, since they end in a plot, my main thought was just to make sure the functions can run all the way through without erroring.  I wasn't sure how to confirm if the plot is "right"

Reply:
I can add the test for plot_farm_terrain, then merge you think if I do it tonight?

Reply:
Yep sounds good to me.

Reply:
Ok great, done and merging!